deploy/templates/nstemplatetiers/advanced/cluster.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  labels:
+    toolchain.dev.openshift.com/provider: codeready-toolchain
+  name: advanced-cluster-resources
+objects:
+- apiVersion: quota.openshift.io/v1
+  kind: ClusterResourceQuota
+  metadata:
+    name: for-${USERNAME}
+  spec:
+    quota:
+      hard:
+        limits.cpu: 1750m
+        limits.memory: 7Gi
+        requests.storage: 5Gi
+        persistentvolumeclaims: "2"
+    selector:
+      annotations:
+        openshift.io/requester: ${USERNAME}
+      labels: null
```

Comment:
These annotation is set in our templates. So, I think it's OK to relay on something which is defined in our templates from another related template.
Why using these annotations in the first place? This is what is used in the OpenShift docs (because all namespaces created via `ProjectRequest` have that annotation) :)
But I'm OK with switching to owner label too.

Reply:
What is important here is to eventually improve our e2e tests which will not only check that the resources created. We should actually check that these resources gives us what we want:
- The limits are actually enforced (it fails if we try to create pods and other resources if they exceed the limits).
- Users can't do stuff outside of expected roles
- etc.
IMO it's very important but will take awhile to implement. So, we can create a separate JIRA for that.